metals/src/main/scala/scala/meta/internal/metals/watcher/SimpleFileWatcher.scala

```diff
@@ -0,0 +1,66 @@
+package scala.meta.internal.metals.watcher
+
+import collection.JavaConverters._
+import java.nio.file.{FileSystems, Path, StandardWatchEventKinds}
+import java.util.concurrent.BlockingQueue
+import scala.concurrent.{ExecutionContext, Future}
+
+object SimpleFileWatcher {
+  /*
+   * returns a function which closes the file watcher when called
+   */
+  def initWatcher(
+    watchFilter: Path => Boolean,
+    watchEventQueue: BlockingQueue[FileWatcherEvent],
+    filesToWatch: Set[Path],
+    directoriesToWatch: Set[Path],
+  )(implicit ec: ExecutionContext): () => Unit = {
+    scribe.error(s"RFERGUSON INIT WATCHER")
```

Comment:
Did you mean to remove this? If not, could we use `scribe.info` instead and make the message more informative?

Reply:
Thank you!